esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -3859,6 +3860,12 @@ void CWsWuFileHelper::createZAPWUQueryAssociatedFiles(IConstWorkUnit* cwu, const
         IConstWUAssociatedFile& cur = iter->query();
         cur.getName(name);
         cur.getIp(ip);
+        if (cur.getType() == FileTypePostMortem)
+        {
+            assert(streq(ip.str(), "localhost"));
+            postMortemFolder.set(name.str());
```

Comment:
see previous comment. I think createZAPWUQueryAssociatedFiles should be reworked, so as to pick up all local files (which will likely be all of them in cloud for the foreseeable future).